examples/clog_examples.clog_config

```diff
@@ -21,15 +24,21 @@
       "EncodedPrefix": null,
       "EncodedArgNumber": 1,
       "MacroConfiguration": {
-        "linux": "lttng_plus"
+        "linux": "lttng_plus",
+        "stubs": "empty",
```

Comment:
Let's rename this to macos and update the cmake accordingly.

Reply:
Well occasionally we use stubs from windows in msquic if logging is disabled. I can add a macos specific one though.